### PR TITLE
Adding ClarityClan Presence

### PR DESCRIPTION
**Pull request by leojacondev**
Hey, please add ClarityClan Presence to store. It's a Minecraft Clan Website.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,5 +31,9 @@
     "semver": "^7.3.2",
     "source-map-support": "^0.5.19",
     "typescript": "^3.9.5"
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
+    "fetch": "^1.1.0"
```
Comment (Alanexei) [inline package.json:37]: Don't modify the `package.json`, if you choose to make HTTP requests use the fetch function.

File: websites/C/ClarityClan/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Leonardo Jacon#0001",
+      "id": "600148454668369930"
+    },
+    "url": "clarityclan.team",
+    "description": {
+      "en": "Clarity Clan is a Minecraft Clan, growing in the community of MCC."
+    },
+    "service": "Clarity Clan",
```
Comment (Alanexei) [inline websites/C/ClarityClan/dist/metadata.json:11]: The service name must be the same as your directory name. Either change this to `Clarity Clan` or change the name of the directory.

File: websites/C/ClarityClan/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/LGdKN10.png",
+    "thumbnail": "https://i.imgur.com/LGdKN10.png",
```
Comment (Alanexei) [inline websites/C/ClarityClan/dist/metadata.json:14]: The thumbnail must not be the same as the logo, take a screenshot of the index page and use that instead.

**Comment by leojacondev** [comment]
Screenshot:
image

**Comment by leojacondev** [comment]
I'll re-fork it and open a new pull request.

**Pull request closed without merging**